fbw-a32nx/src/systems/instruments/src/EFB/Ground/Pages/Payload/A320_251N/A320Payload.tsx

```diff
@@ -452,6 +460,23 @@ export const A320Payload: React.FC<A320Props> = ({
         gsxPayloadSyncEnabled,
     ]);
 
+    // If totalPax is 0, deactivate all sounds from the cabin.
+    useEffect(() => {
+        const cabinSoundStatus:number = (totalPax > 0 ? 1 : 0);
```

Comment:
Are you specifying this is a `number` here since TypeScript is being even more explicit and inferring `cabinSoundStatus` (without any help) to be either `1 | 0`?

I feel that what TypeScript is coming up with here is indeed correct, since those are the only two options - but if you think that `number` is a better type, then let's stick with that!

Reply:
In the simVars the sounds are stored as 0 = disabled and 1 = enabled. CabinSoundStatus is the value which will be wrote in the simVar later, so it should be 1 or 0.